src/types/amazon.ts

```diff
@@ -1,7 +1,7 @@
-/**
- * From: https://developer.amazon.com/es/docs/in-app-purchasing/iap-rvs-examples.html
- */
-export type ReceiptType = {
+import type {Sku} from './index';
```

Comment:
Not sure I fully understand why use the `Sku` type alias, is there a benefit vs just using string?

Reply:
Hum, theoretically, if the type of Sku were to change, it would be easy and be done in one place, instead of going over all the `sku: string`. In reality, that probably won't happen, but it just to give some extra type safety and define the type only in one place to make it clear.